### PR TITLE
Created all neccesery classes and methods

### PR DESCRIPTION
**Pull request by shafran-andriy**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -2,4 +2,66 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        if coords is None:
```
Comment (vasylhnatiuk) [inline app/main.py:8]: do it in one line

File: app/main.py [hunk 2]
```diff
@@ -2,4 +2,66 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
```
Comment (vasylhnatiuk) [inline app/main.py:7]:
```suggestion
    def __init__(self, name: str, weight: int, coords: list[int] = None) -> None:
```

Comment (vasylhnatiuk) [reply app/main.py:7]: fix everywhere

File: app/main.py (hunk 1 above)
Comment (shafran-andriy) [reply app/main.py]: fixed

File: app/main.py (hunk 2 above)
Comment (shafran-andriy) [reply app/main.py]: fixed

File: app/main.py [hunk 3]
```diff
@@ -2,4 +2,72 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: list[int] = None
```
[the remark this answers is not in the document]
Comment (shafran-andriy) [reply app/main.py]: Fixed

**Review by EdAlekseiev — APPROVED** [review]

**Pull request closed without merging**